diff --git a/node_modules/react-native-safe-area-context/lib/commonjs/SafeAreaView.js b/node_modules/react-native-safe-area-context/lib/commonjs/SafeAreaView.js
index d7f1182..45edc33 100644
--- a/node_modules/react-native-safe-area-context/lib/commonjs/SafeAreaView.js
+++ b/node_modules/react-native-safe-area-context/lib/commonjs/SafeAreaView.js
@@ -6,43 +6,13 @@ Object.defineProperty(exports, "__esModule", {
 exports.SafeAreaView = void 0;
 var _react = _interopRequireWildcard(require("react"));
 var React = _react;
-var _NativeSafeAreaView = _interopRequireDefault(require("./specs/NativeSafeAreaView"));
-function _interopRequireDefault(e) { return e && e.__esModule ? e : { default: e }; }
+var _reactNative = require("react-native");
 function _interopRequireWildcard(e, t) { if ("function" == typeof WeakMap) var r = new WeakMap(), n = new WeakMap(); return (_interopRequireWildcard = function (e, t) { if (!t && e && e.__esModule) return e; var o, i, f = { __proto__: null, default: e }; if (null === e || "object" != typeof e && "function" != typeof e) return f; if (o = t ? n : r) { if (o.has(e)) return o.get(e); o.set(e, f); } for (const t in e) "default" !== t && {}.hasOwnProperty.call(e, t) && ((i = (o = Object.defineProperty) && Object.getOwnPropertyDescriptor(e, t)) && (i.get || i.set) ? o(f, t, i) : f[t] = e[t]); return f; })(e, t); }
 function _extends() { return _extends = Object.assign ? Object.assign.bind() : function (n) { for (var e = 1; e < arguments.length; e++) { var t = arguments[e]; for (var r in t) ({}).hasOwnProperty.call(t, r) && (n[r] = t[r]); } return n; }, _extends.apply(null, arguments); }
-const defaultEdges = {
-  top: 'additive',
-  left: 'additive',
-  bottom: 'additive',
-  right: 'additive'
-};
 const SafeAreaView = exports.SafeAreaView = /*#__PURE__*/React.forwardRef(({
-  edges,
+  edges: _edges,
   ...props
-}, ref) => {
-  const nativeEdges = (0, _react.useMemo)(() => {
-    if (edges == null) {
-      return defaultEdges;
-    }
-    const edgesObj = Array.isArray(edges) ? edges.reduce((acc, edge) => {
-      acc[edge] = 'additive';
-      return acc;
-    }, {}) :
-    // ts has trouble with refining readonly arrays.
-    edges;
-
-    // make sure that we always pass all edges, required for fabric
-    const requiredEdges = {
-      top: edgesObj.top ?? 'off',
-      right: edgesObj.right ?? 'off',
-      bottom: edgesObj.bottom ?? 'off',
-      left: edgesObj.left ?? 'off'
-    };
-    return requiredEdges;
-  }, [edges]);
-  return /*#__PURE__*/React.createElement(_NativeSafeAreaView.default, _extends({}, props, {
-    edges: nativeEdges,
+}, ref) => /*#__PURE__*/React.createElement(_reactNative.View, _extends({}, props, {
     ref: ref
-  }));
-});
+  })));
 //# sourceMappingURL=SafeAreaView.js.map
diff --git a/node_modules/react-native-safe-area-context/lib/module/SafeAreaView.js b/node_modules/react-native-safe-area-context/lib/module/SafeAreaView.js
index 3f17805..8057fff 100644
--- a/node_modules/react-native-safe-area-context/lib/module/SafeAreaView.js
+++ b/node_modules/react-native-safe-area-context/lib/module/SafeAreaView.js
@@ -1,40 +1,10 @@
 function _extends() { return _extends = Object.assign ? Object.assign.bind() : function (n) { for (var e = 1; e < arguments.length; e++) { var t = arguments[e]; for (var r in t) ({}).hasOwnProperty.call(t, r) && (n[r] = t[r]); } return n; }, _extends.apply(null, arguments); }
 import * as React from 'react';
-import NativeSafeAreaView from './specs/NativeSafeAreaView';
-import { useMemo } from 'react';
-const defaultEdges = {
-  top: 'additive',
-  left: 'additive',
-  bottom: 'additive',
-  right: 'additive'
-};
+import { View } from 'react-native';
 export const SafeAreaView = /*#__PURE__*/React.forwardRef(({
-  edges,
+  edges: _edges,
   ...props
-}, ref) => {
-  const nativeEdges = useMemo(() => {
-    if (edges == null) {
-      return defaultEdges;
-    }
-    const edgesObj = Array.isArray(edges) ? edges.reduce((acc, edge) => {
-      acc[edge] = 'additive';
-      return acc;
-    }, {}) :
-    // ts has trouble with refining readonly arrays.
-    edges;
-
-    // make sure that we always pass all edges, required for fabric
-    const requiredEdges = {
-      top: edgesObj.top ?? 'off',
-      right: edgesObj.right ?? 'off',
-      bottom: edgesObj.bottom ?? 'off',
-      left: edgesObj.left ?? 'off'
-    };
-    return requiredEdges;
-  }, [edges]);
-  return /*#__PURE__*/React.createElement(NativeSafeAreaView, _extends({}, props, {
-    edges: nativeEdges,
+}, ref) => /*#__PURE__*/React.createElement(View, _extends({}, props, {
     ref: ref
-  }));
-});
+  })));
 //# sourceMappingURL=SafeAreaView.js.map
diff --git a/node_modules/react-native-safe-area-context/src/SafeAreaView.tsx b/node_modules/react-native-safe-area-context/src/SafeAreaView.tsx
index 1fe6b80..5f03888 100644
--- a/node_modules/react-native-safe-area-context/src/SafeAreaView.tsx
+++ b/node_modules/react-native-safe-area-context/src/SafeAreaView.tsx
@@ -1,50 +1,13 @@
 import * as React from 'react';
+import { View } from 'react-native';
 import type {
-  Edge,
-  EdgeMode,
-  EdgeRecord,
   NativeSafeAreaViewInstance,
   NativeSafeAreaViewProps,
 } from './SafeArea.types';
-import NativeSafeAreaView from './specs/NativeSafeAreaView';
-import { useMemo } from 'react';
-
-const defaultEdges: Record<Edge, EdgeMode> = {
-  top: 'additive',
-  left: 'additive',
-  bottom: 'additive',
-  right: 'additive',
-};
 
 export type SafeAreaViewProps = NativeSafeAreaViewProps;
 
 export const SafeAreaView = React.forwardRef<
   NativeSafeAreaViewInstance,
   SafeAreaViewProps
->(({ edges, ...props }, ref) => {
-  const nativeEdges = useMemo(() => {
-    if (edges == null) {
-      return defaultEdges;
-    }
-
-    const edgesObj = Array.isArray(edges)
-      ? edges.reduce<EdgeRecord>((acc, edge: Edge) => {
-          acc[edge] = 'additive';
-          return acc;
-        }, {})
-      : // ts has trouble with refining readonly arrays.
-        (edges as EdgeRecord);
-
-    // make sure that we always pass all edges, required for fabric
-    const requiredEdges: Record<Edge, EdgeMode> = {
-      top: edgesObj.top ?? 'off',
-      right: edgesObj.right ?? 'off',
-      bottom: edgesObj.bottom ?? 'off',
-      left: edgesObj.left ?? 'off',
-    };
-
-    return requiredEdges;
-  }, [edges]);
-
-  return <NativeSafeAreaView {...props} edges={nativeEdges} ref={ref} />;
-});
+>(({ edges: _edges, ...props }, ref) => <View {...props} ref={ref} />);
